src/commands/ml-models-create.ts

```diff
@@ -0,0 +1,22 @@
+import { Command } from "../command";
+import { FirebaseModel } from "../ml/models";
+
+export default new Command("ml:models:create <displayName> [source]")
+  .description(
```

Comment:
(I should add this to the contributing guide:)

`.description` should be a one-line description. `.help` can be used to provide more verbose text about how the command should be used.

Reply:
Ok, I'll shorten the description and add the .help string later with the help of our docs person.